src/fragments/lib/push-notifications/common/enable-rich-notifications.mdx

```diff
@@ -33,13 +33,13 @@ target 'MyNotificationServiceExtension' do # Replace with your service extension
 end
 ```
 
-3. Install the new pods by running the following command at the root of your React Native project.
+3. Install the new pods by running the following command at the root of your application project.
 
 ```bash
 npx pod-install
```

Comment:
Might need a flutter fragment 😅 

```bash
pod install
```

Also need to add a note about installing cocoapods to make `pod` command available?